ods_ci/tasks/Resources/RHODS_OLM/install/install.robot

```diff
@@ -1,3 +1,11 @@
+*** Settings ***
+Documentation    RHODS operator installation top-level keywords
+
+
+*** Variables ***
+${MANAGE_CODEFLARE_BOOL} =    Convert To Boolean    ${MANAGE_CODEFLARE_OPERATOR}
```

Comment:
where `MANAGE_CODEFLARE_OPERATOR` defined?

Reply:
It should be defined in the `deployRhods` pipeline library method here: https://gitlab.cee.redhat.com/ods/jenkins/-/merge_requests/451/diffs#75c1d85aec243f843c64b69364a3b76b88614656_127_131

Not sure if I need to declare/define it in some other place as well?

Reply:
hm well, it depends on the desired final result, meaning:
1. we could decide to set a default value 'FALSE' and if anybody runs the code from local or without setting the variable at runtime, the code works.
2. the code is not supposed to run if the variable don't get set at runtime.